code/scripting/api/objs/control_binding.cpp

```diff
@@ -73,6 +73,19 @@ ADE_FUNC(lock, l_ControlBinding, "boolean lock", "Locks this control binding whe
 	return ADE_RETURN_NIL;
 }
 
+ADE_FUNC(isLock, l_ControlBinding, nullptr, "Locks this control binding when true, disables if false. Persistent between missions.", "boolean lock", "If this control is locked, nil if the handle is invalid")
```

Comment:
Since the actual field is called `locked` it would be appropriate to call this `isLocked`.

Also, the description makes it seem like this is changing something while it simply returns a value.